cli/main.go

```diff
@@ -1,15 +1,13 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
 	"github.com/replicatedhq/replicated/cli/cmd"
 )
 
 func main() {
 	if err := cmd.Execute(nil, os.Stdin, os.Stdout, os.Stderr); err != nil {
-		fmt.Println(err)
```

Comment:
Fixes double-printing of error messages (as in https://github.com/spf13/cobra/issues/304)